test/library/packages/ArgumentParser/doc-examples/ArgumentParserExamples.chpl

```diff
@@ -0,0 +1,61 @@
+//
+// THIS TEST CASE IS AUTO-INCLUDED IN THE DOCUMENTATION
+//
+
+use ArgumentParser;
+
+/* START_EXAMPLE_0 */
+prototype module M {
```

Comment:
this example is different than what was in the docs. I am cautious about introducing the concept of `prototype` modules to this example,